src/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteProperties.String.cs

```diff
@@ -494,24 +488,18 @@ public void WriteString(ReadOnlySpan<byte> utf8PropertyName, ReadOnlySpan<byte>
         /// <summary>
         /// Writes the pre-encoded property name and string text value (as a JSON string) as part of a name/value pair of a JSON object.
         /// </summary>
-        /// <param name="propertyName">The JSON encoded property name of the JSON object to be transcoded and written as UTF-8.</param>
-        /// <param name="value">The value to be written as a UTF-8 transcoded JSON string as part of the name/value pair.</param>
-        /// <remarks>
-        /// <para>
-        /// The property name should already be escaped when the instance of <see cref="JsonEncodedText"/> was created.
-        /// The value is escaped before writing.
```

Comment:
We need this remark. `The value is escaped before writing`. The property name isn't.

Reply:
Thanks, must have been from the merge conflicts. Fixed.